app/main.py

```
@@ -1,6 +1,26 @@
+import os
+import requests
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.environ.get("API_KEY")
+    if not api_key:
+        raise ValueError("API_KEY not found in environment variables")
+
+    url = "http://api.weatherapi.com/v1/current.json"
+    params = {"key": api_key, "q": "Paris"}
+
+    response = requests.get(url, params=params)
+    response.raise_for_status()
+    data = response.json()
```

Behavior requirement: the code uses response.raise_for_status() but does not catch exceptions or provide a friendly error message. This violates the requirement to: "Handle unsuccessful HTTP responses from the Weather API by detecting non-2xx status codes and providing some error output rather than failing silently or crashing without message." Wrap the request/response code in try/except (catch `requests.exceptions.RequestException` or `requests.exceptions.HTTPError`), print a helpful message including status code/response text when possible, and exit with a non-zero code on failure.